SpatialGDK/Source/SpatialGDKServices/Private/LocalDeploymentManager.cpp

```diff
@@ -14,7 +14,7 @@
 
 DEFINE_LOG_CATEGORY(LogSpatialDeploymentManager);
 
-static const FString SpatialServiceVersion(TEXT("20190716.094149.1b6d448edd"));
+static const FString SpatialServiceVersion(TEXT("20190930.180414.3b04a59226"));
```

Comment:
Does SpatialOutputLog require this specific SpatialServiceVersion?

Reply:
Yeah, it's spatiald which actually prints the launch.log file.